### PR TITLE
Evolve and docs.

### PR DESCRIPTION
**Pull request by Tinche**
✨

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,6 +8,15 @@ The third digit is only for regressions.
 16.4.0 (UNRELEASED)
```
Comment (hynek) [inline CHANGELOG.rst]: (make it 17.1.0 pls, I’ve stopped doing .0.0 releases this year)

**Comment by Tinche** [comment]
Sure thing, the changes will be done tonight.

**Comment by Tinche** [comment]
Done!

**Comment by hynek** [comment]
Awesome, thanks!

**Pull request merged**